### PR TITLE
Directional matches

### PR DESCRIPTION
**Pull request by johnameyer**

### CODE REVIEW

File: app/src/views/Matches.vue [hunk 1]
```diff
@@ -5,94 +5,22 @@
...
-              </v-card>
-            </div>
-          </div>
-          <div v-else>Could not load matches</div>
-        </div>
-      </v-fade-transition>
+      <v-tabs dark v-model="tab" icons-and-text>
+        <v-tab>Two way matches<v-icon>mdi-account-switch</v-icon></v-tab>
+        <v-tab>People who have matched back<v-icon>mdi-account-arrow-left</v-icon></v-tab>
+        <v-tab>People you've matched with<v-icon>mdi-account-arrow-right</v-icon></v-tab>
```
Comment (johnameyer) [inline app/src/views/Matches.vue:11]: @rkarim0598 wasn't sure what to go with on these names

File: server/db/pool.js [hunk 2]
```diff
@@ -7,5 +7,11 @@ module.exports = async function() {
     if(pool == null) {
         pool = await oracledb.createPool(config);
     }
-    return pool.getConnection();
+    const connection = pool.getConnection();
+    return connection.then((connection) => {
+        setTimeout(async () => {
+            try { await connection.ping(); await connection.break(); } catch {}
+        }, 3000);
+        return connection;
```
Comment (johnameyer) [inline server/db/pool.js:15]: @rkarim0598 this is my current amelioration to the problem, it doesn't work well in the long term for some reason I believe

File: app/src/views/Matches.vue (hunk 1 above)
Comment (rkarim0598) [reply app/src/views/Matches.vue:11]: Maybe "People you like" and "People who like you"?

File: app/src/views/Matches.vue [hunk 3]
```diff
@@ -5,94 +5,22 @@
...
+        <v-tab-item>
+          <match-list :query="$apollo.queries.matches" :matches="matches"></match-list>
+        </v-tab-item>
+        <v-tab-item>
+          <match-list :query="$apollo.queries.withYou" :matches="withYou"></match-list>
+        </v-tab-item>
+        <v-tab-item>
+          <match-list :query="$apollo.queries.fromYou" :matches="fromYou"></match-list>
+        </v-tab-item>
+      </v-tabs-items>
```
Comment (rkarim0598) [inline app/src/views/Matches.vue:23]: Such beauty I have not seen

**Pull request merged**